src/redux/restReducers/sendMelding.ts

```diff
@@ -1,11 +1,49 @@
 import { apiBaseUri } from '../../api/config';
 import { AppState } from '../reducers';
-import { Melding } from '../../models/meldinger/meldinger';
+import { SendMeldingRequest } from '../../models/meldinger/meldinger';
 import createPostResourceReducerAndActions from '../../rest/utils/postResource';
+import { AsyncDispatch } from '../ThunkTypes';
+import { getSaksbehandlerIdent } from '../../utils/loggInfo/getSaksbehandlerIdent';
+import { getSaksbehandlerEnhet } from '../../utils/loggInfo/saksbehandlersEnhetInfo';
+import { loggError } from '../../utils/frontendLogger';
 
 export function getMeldingPostUri(state: AppState): string {
     const fnr = state.gjeldendeBruker.fødselsnummer;
-    return `${apiBaseUri}/dialog/${fnr}/meldinger/sendmelding`;
+    return `${apiBaseUri}/dialog/${fnr}/sendmelding`;
 }
 
-export default createPostResourceReducerAndActions<Melding>('meldinger', getMeldingPostUri);
+export const resetSendMeldingActionCreator = (dispatch: AsyncDispatch, getState: () => AppState) =>
+    dispatch(getState().restResources.sendMelding.actions.reset);
+
+export function sendMeldingActionCreator(
+    payload: Pick<SendMeldingRequest, 'fritekst' | 'kanal' | 'type' | 'temagruppe' | 'traadId' | 'kontorsperretEnhet'>
+) {
+    return (dispatch: AsyncDispatch, getState: () => AppState) => {
+        const saksbehandler = getSaksbehandlerIdent();
+        const state = getState();
+        if (!saksbehandler) {
+            const error = new Error('Kunne ikke finne saksbehandlerident');
+            loggError(error);
+            dispatch(state.restResources.sendMelding.actions.setError(error));
```

Comment:
Er det ikke veldig merksnodig å hente ut `actions` fra `state`?? :/

Reply:
hehe, jo, men vi ble enige en stund tilbake om at det ble ganske ryddig, så da gikk vi for det mønsteret. Kan godt vise deg senere :-)